lib/quiz/quiz_pager.dart

```diff
@@ -1,24 +1,32 @@
 import 'dart:convert';
 import 'package:flutter/material.dart';
+import 'package:maui/components/hud.dart';
 import 'package:maui/games/single_game.dart';
 import 'package:maui/repos/quiz_repo.dart';
 import 'package:maui/db/entity/quiz.dart';
 import 'match_the_following.dart';
 import 'multiple_choice.dart';
 import 'grouping_quiz.dart';
+import 'quiz_scroller_pagger.dart';
 import 'true_or_false.dart';
 import 'sequence.dart';
 import 'quiz_result.dart';
 
+
+
 class QuizPager extends StatefulWidget {
   Function onScore;
   Function onProgress;
   Function onEnd;
   Function onTurn;
   int iteration;
-  GameConfig gameConfig;
+ GameConfig gameConfig;
```

Comment:
Pay attention to proper indentation. It is just a matter of asking the editor (VSCode) to do it for you.